frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-storageclass-selection.spec.ts

```diff
@@ -28,6 +35,10 @@ const addCapacity = (uid: string, scName: string) => {
   cy.byTestID('dropdown-menu-item-link')
     .contains(scName)
     .click();
+
+  // wait for Pvs & Nodes data to load (used for validations)
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(5000);
```

Comment:
This can be a source of flakiness. We should try to avoid waits, since cypress by itself waits for components to be visible.

Reply:
added a to-do task foe later...